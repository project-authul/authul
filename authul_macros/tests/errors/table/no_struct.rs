#[authul_macros::authul_table]
