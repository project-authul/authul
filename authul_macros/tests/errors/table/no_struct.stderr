error: expected item after attributes
 --> tests/errors/table/no_struct.rs:1:1
  |
1 | #[authul_macros::authul_table]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
